base_phone/base_phone.py

```diff
@@ -77,7 +77,7 @@ def _generic_reformat_phonenumbers(
                 if user.company_id.country_id:
                     countrycode = user.company_id.country_id.code
                 else:
-                    _logger.error(
+                    _logger.warning(
                         _("You should set a country on the company '%s' "
                             "to allow the reformat of phone numbers")
```

Comment:
My point was only to change the level, but ok if you want. I replaced the `%` by `,` and removed the gettext call, a log should not be translated.